core/src/test/java/io/questdb/test/jit/CompiledFilterIRSerializerTest.java

```diff
@@ -847,9 +847,16 @@ public void testInSubSelect() throws Exception {
         serialize("asymbol in (select asymbol from tab limit 1)");
     }
 
-    @Test(expected = SqlException.class)
+    @Test
     public void testTimestampInLiteral() throws Exception {
```

Comment:
Please add a test with bind variables, e.g.
```java
        bindVariableService.clear();
        bindVariableService.setStr("str", "2020");
        serialize("atimestamp in :str");
```

Such filters shouldn't be JIT compiled since interval parsing has to be done in runtime.

Reply:
Done in last commit